extensions/wikia/CommunityPage/tests/WikiTopicTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class WikiTopicTest extends WikiaBaseTest {
+
+	public function setUp() {
+		$this->setupFile = __DIR__ . '/../CommunityPage.setup.php';
+		parent::setUp();
+	}
+	/**
+	 * @dataProvider FormingWikiTopicProvider
+	 */
+	public function testFormingWikiTopic( $sitename, $expectedWikiTopic ) {
+		$this->mockGlobalVariable( 'wgWikiTopic', null );
```

Comment:
You could add a value to this to the data provider and also include a test for if `wgWikiTopic` is set and so overrides `wgSitename`.